pkg/rest/core/fake/rest_client.go

```diff
@@ -375,7 +375,7 @@ func deleteItem(storage NamespacedStorage) func(http.ResponseWriter, *http.Reque
 			return
 		}
 		storage.Delete(ns, tipe, name)
-		rw.WriteHeader(http.StatusOK)
+		rw.WriteHeader(http.StatusAccepted)
```

Comment:
I guess this is trying to match whatever upstream does?

I think it makes sense in the 'this is not a synchronous call' idea, but I'm not sure where it's documented. A quick look at a couple of the upstream api docs doesn't say anything specific.